vasyl_varyshchuk/Week_1/task_1/max_of_list.py

```diff
@@ -0,0 +1,28 @@
+'''
+This module finds maximum integer value from list of strings.
+
+'''
+
+data = ['one', '1', 'two', '2', 'ten', '10', '15', '-2', 'some long text goes here', '16.9', '&']
+
+def get_max_value(iterable):
```

Comment:
Please fix your code according to PEP-8:
https://www.python.org/dev/peps/pep-0008/#blank-lines

Some additional information might be found here:
https://stackoverflow.com/a/2953268